fmt/testdata/FunctionCallArgsStatement/fmt.sol

```diff
@@ -0,0 +1,53 @@
+// config: bracket-spacing=true
+interface ITarget {
+    function run() external payable;
+    function veryAndVeryLongNameOfSomeRunFunction() external payable;
+}
+
+contract FunctionCallArgsStatement {
+    ITarget public target;
+
+    function estimate() public returns (uint256 gas) {
+        gas = 1 gwei;
+    }
+
+    function veryAndVeryLongNameOfSomeGasEstimateFunction()
+        public
+        returns (uint256)
+    {
+        return gasleft();
+    }
+
+    function value(uint256 val) public returns (uint256) {
+        return val;
+    }
+
+    function test() external {
+        target.run{ gas: gasleft(), value: 1 wei };
```

Comment:
I think we want this without the leading space? cc @mds1 

```suggestion
        target.run{gas: gasleft(), value: 1 wei };
```

Reply:
let's go with no space and add the space if more than 3 people complain

Reply:
no leading space but still closing space? thats pretty weird imo

Reply:
makes sense if there's a trailing comma but there's not?

Reply:
no closing space either sorry